### ISSUE TITLE
Option to specify path

### ISSUE THREAD

**Issue by jbussemaker** [capaulson/pyKriging#27]
First of all, thank you for your package! Apart from 2 issues I found (of which this is one), everything works very well!

I was getting a IOError when generating an optimal LHC sampling plan using the samplingplan class, because it was trying to write the LHC pickled cache to its installation directory (I'm running Anaconda2 on Windows), to which I didn't have writing rights. I think that in general it can be assumed, especially for large corporate system architectures, that Python has no admin rights and thus cannot write to the installation directory of the package. It would be nice if the path could be supplied as a parameter to the __init__ function to solve this issue.

**Comment by capaulson** [comment]
@jbussemaker yeah, I've thought about ways to solve this. So far I've come up short with a fully general solution. This needs to work on Mac, Linux and Windows. It needs to build from source, install from pip and things like Anaconda. I would appriciate any guidence from the comunity on how best to address this.

**Comment by jbussemaker** [comment]
@capaulson what about appdirs (https://pypi.org/project/appdirs/)? That one should offer the cross-platform user-directory approach that you need.
